test/onnx/test_fx_to_onnx.py

```diff
@@ -172,6 +173,39 @@ def forward(self, input):
             expected_error_message="ValueError",
         )
 
+    def test_symbolic_tracing_simple(self):
+        from torch._subclasses import fake_tensor
+        from torch.fx.experimental.symbolic_shapes import ShapeEnv
+
+        class Model(torch.nn.Module):
+            def __init__(self) -> None:
+                super().__init__()
+                self.linear = torch.nn.Linear(2, 2)
+
+            def forward(self, x):
+                out = self.linear(x)
+                return out
+
+        # User-instantiated FakeTensorMode
+        fake_mode = fake_tensor.FakeTensorMode(
+            allow_non_fake_inputs=True,
+            shape_env=ShapeEnv(
```

Comment:
Following up on the previous `frame_id` comment, I was coming from a point if user should be required to set these things themselves. 

Do configurations here override or gets overridden by dynamo.export call later on? I think either way this is incentive for us providing API to hideaway this detail by default.

Reply:
No, if a `fake_mode` is provided to `torch._dynamo.export` (see #100017), it is reused as-is.

I am Ok in refactoring this API when the whole symbolic tracing is working in all scenarios we care about.